lib/src/peer.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:async';
 
+import 'package:pedantic/pedantic.dart';
```

Comment:
Our dependency on `pedantic` is currently in `dev_dependencies`. This is a "core" enough package that I think it might be smoother to keep it there instead of `dependencies`. We do sometimes write less idiomatic code for that reason.

How about instead we do something like `return _manager.listen((mesage) {...}).whenComplete(close);` and drop the `async` keyword so we don't need to use `unawaited` here.

Reply:
Done!